test/publish_test.js

```diff
@@ -10,7 +10,7 @@ suite('api/publish', function() {
   if (!cfg.aws || !cfg.referenceTestBucket) {
     console.log('Skipping \'publish\', missing config file: ' +
                 'taskcluster-base-test.conf.json');
-    this.pending = true;
+  this.skip = true;
```

Comment:
`skip` is a method; `pending` is a property.  I don't know why, but that's how mocha works.  Anyway, setting `skip = true` won't do anything.